code/modules/mob/living/simple_animal/hostile/hostile.dm

```diff
@@ -5,7 +5,7 @@
 	environment_smash = ENVIRONMENT_SMASH_STRUCTURES //Bitflags. Set to ENVIRONMENT_SMASH_STRUCTURES to break closets,tables,racks, etc; ENVIRONMENT_SMASH_WALLS for walls; ENVIRONMENT_SMASH_RWALLS for rwalls
 	var/atom/target
 	var/ranged = 0
-	var/rapid = 0
+	var/rapid = 0 //shoots 3 projectiles if 1, and 6 projectiles if 2
```

Comment:
This kind of thing should use a define instead of magic numbers.
Something like
```
#define HOSTILE_PROJECTILES_ONE 0
#define HOSTILE_PROJECTILES_THREE 1
#define HOSTILE_PROJECTILES_6 3
```